requirements.txt

```diff
@@ -12,7 +11,7 @@ onnxruntime==1.15.0; sys_platform == 'darwin' and platform_machine != 'arm64'
 onnxruntime-silicon==1.13.1; sys_platform == 'darwin' and platform_machine == 'arm64'
 onnxruntime-gpu==1.15.0; sys_platform != 'darwin'
 tensorflow==2.13.0rc1; sys_platform == 'darwin'
-tensorflow==2.12.0; sys_platform != 'darwin'
+tensorflow==2.13.0rc1; sys_platform != 'darwin'
```

Comment:
Please undo this

Reply:
It won't be able to install all the dependencies, because `requirements.txt` is broken at the moment. There is a conflict on the required version of transient `numpy`.